src/jit/lower.h

```diff
@@ -269,9 +270,10 @@ class Lowering : public Phase
         return LIR::AsRange(m_block);
     }
 
-    LinearScan* m_lsra;
-    unsigned    vtableCallTemp; // local variable we use as a temp for vtable calls
-    BasicBlock* m_block;
+    LinearScan*   m_lsra;
+    unsigned      vtableCallTemp; // local variable we use as a temp for vtable calls
+    SideEffectSet scratchSideEffects; // SideEffectSet used for IsSafeToContainMem and isRMWIndirCandidate
```

Comment:
The new variables should be prefixed with `m_` to make it clear that they are instance members.